src/Apps/IdentityVerification/__tests__/IdentityVerificationPage.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { IdentityVerificationPageTestQueryRawResponse } from "__generated__/IdentityVerificationPageTestQuery.graphql"
+import { routes_IdentityVerificationPageQueryResponse } from "__generated__/routes_IdentityVerificationPageQuery.graphql"
+import deepMerge from "deepmerge"
+import { createTestEnv } from "DevTools/createTestEnv"
+import { Location } from "found"
+import React from "react"
+import { graphql } from "react-relay"
+import { IdentityVerificationPageQueryResponseFixture } from "../__fixtures__/routes_IdentitifcationPageQuery"
+import IdentityVerificationPage from "../IdentityVerificationPage"
+import { IdentityVerificationTestPage } from "./Utils/IdentityVerificationTestPage"
+
+jest.unmock("react-relay")
+jest.unmock("react-tracking")
+jest.mock("Utils/Events", () => ({
+  postEvent: jest.fn(),
+}))
+const mockLocation: Partial<Location> = {}
+const mockPostEvent = require("Utils/Events").postEvent as jest.Mock
+
+const setupTestEnv = ({
+  location = mockLocation,
+}: {
+  location?: Partial<Location>
+} = {}) => {
+  return createTestEnv({
+    TestPage: IdentityVerificationTestPage,
+    Component: (props: routes_IdentityVerificationPageQueryResponse) => (
+      <IdentityVerificationPage {...props} />
+    ),
+    query: graphql`
+      query IdentityVerificationPageTestQuery @raw_response_type {
+        me {
+          ...IdentityVerificationPage_me @arguments(id: "idv-id")
+        }
+      }
+    `,
+    defaultData: IdentityVerificationPageQueryResponseFixture as IdentityVerificationPageTestQueryRawResponse,
+  })
+}
+
+describe("IdentityVerification route", () => {
+  afterEach(() => {
+    jest.resetAllMocks()
+  })
+  describe("for signed-in user", () => {
+    it("allows an identity instance owner to begin identity verification", async () => {
```

Comment:
#minors:
  - thoughts on changing the docstring to only specify that a user is able to _view_ a page about the identity verification for now, and not begin (at least not yet).
  - thoughts on separating the setup / act / assert phases of this test with newlines. I, personally, think that would make the test code more scan-able.